src/server/fetch.ts

```diff
@@ -388,14 +394,16 @@ function toHeadersArray(rawHeaders: string[]): types.HeadersArray {
 const redirectStatus = [301, 302, 303, 307, 308];
 
 function canSetCookie(cookieDomain: string, hostname: string) {
```

Comment:
Looks very similar to `domainMatches` - perhaps reuse one?

Reply:
Done.